search/lib/api/provider.js

```diff
@@ -16,31 +16,19 @@ async function getProvider (request, response) {
     const isProvider = providerList.filter(providerObj => providerObj['provider-id'] === providerId);
     if (isProvider.length === 0) throw new Error(`Provider [${providerId}] not found`);
 
-    let providerHoldings, id, rootCatalogName;
+    let rootCatalogName = 'CMR-STAC Root catalog';
```

Comment:
isn't the scope of let defined to the enclosing {} where var is the enclosing function? This is set inside the try{} but used outside it, is this a problem?

Reply:
You are right that it would be a problem, but I don't think that was the case here, also the var has been removed so it's moot.